### ISSUE TITLE
[13.0 -> 14.0][account]unsupported format character '(' (0x28) at index 3

### ISSUE THREAD

**Issue by jprokop1** [OCA/OpenUpgrade#3028]
Hi, 
my problem is while migration ver. 13 -> 14, in module account

Steps I've done:

uninstall all custom modules
create migration enviroment as by https://oca.github.io/OpenUpgrade/
run migration
 

error log:
```
2021-12-26 18:31:50,816 2442 ERROR H13MIG14C OpenUpgrade: account: error in migration script /openupgrade14/OpenUpgrade/openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py: unsupported format character '(' (0x28) at index 3
2021-12-26 18:31:50,816 2442 ERROR H13MIG14C OpenUpgrade: unsupported format character '(' (0x28) at index 3
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/openupgradelib/openupgrade.py", line 1877, in wrapped_function
    func(
  File "/openupgrade14/OpenUpgrade/openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py", line 747, in migrate    fill_statement_lines_with_no_move(env)
  File "/openupgrade14/OpenUpgrade/openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py", line 452, in fill_statement_lines_with_no_move
    st_line.move_id = move
  File "/odoo14/odoo14-server/odoo/fields.py", line 1129, in __set__
    records.write({self.name: write_value})
  File "/odoo14/odoo14-server/addons/account/models/account_bank_statement.py", line 874, in write
    res = super().write(vals)
  File "/odoo14/odoo14-server/odoo/models.py", line 3721, in write
    self._check_company()
  File "/odoo14/odoo14-server/odoo/models.py", line 3318, in _check_company
    lines.append(msg % {
ValueError: unsupported format character '(' (0x28) at index 3
```

is the problem in data or in code? 

**Additional context**
Ubuntu 20.04, Python 3.8.10

Thanks

**Comment by pedrobaeza** [comment]
Please put a print or something similar before such line to know the contents of the `vals` dictionary.

**Comment by jprokop1** [comment]
![image](https://user-images.githubusercontent.com/46556236/147421460-18d5714b-60fa-401e-a26e-aa091409285e.png)
is this enough?

Thanks

**Comment by jprokop1** [comment]
Finally the migration was successfull, but only after deactivation slovak language.
Now the log looks like this (the same section):
 
![image](https://user-images.githubusercontent.com/46556236/147596324-7ba7222c-c64d-4c59-bb2e-1bcf0af51e88.png)
I guess the problem may be in incorrect translation, but where to look?
Thanks

**Comment by jprokop1** [comment]
Hi,

after more investigation I found the reason:
problems are caused by records from account_bank_statement_line (in fill_statement_lines_with_no_move), where partner_id is not null

The whole migration was from v12 to v14, multicompany (3) database with common contacts  but separated products and product categories.
In v13 the record rule for contacts was changed. 

After deleting the assignment of contacts to companies in v13 the migration to v14 was successfull.

I close the issue
Thanks
best regards
